packages/components/popover/package.json

```
@@ -23,5 +23,8 @@
   },
   "dependencies": {
     "@radix-ui/react-popover": "1.0.6"
+  },
+  "devDependencies": {
+    "jsdom-testing-mocks": "^1.9.0"
```

```suggestion
    "jsdom-testing-mocks": "1.9.0"
```
Please see our last [agreements](https://github.com/adevinta/spark/wiki/Agreements#managing-dependencies-within-spark-components) :+1: